Sources/Build/BuildManifest/LLBuildManifestBuilder+Swift.swift

```diff
@@ -43,7 +43,13 @@ extension LLBuildManifestBuilder {
         // Outputs.
         let objectNodes = try target.objects.map(Node.file)
         let moduleNode = Node.file(target.moduleOutputPath)
-        let cmdOutputs = objectNodes + [moduleNode]
+        let cmdOutputs: [Node]
+        if target.prepareForIndexing {
+            // Don't include the object nodes on prepare builds
+            cmdOutputs = [moduleNode]
+        } else {
+            cmdOutputs = objectNodes + [moduleNode]
```

Comment:
Have you considered adjusting `objects` to produce `[]` if the target is in "preparation" mode?

Reply:
Good point. I've been slowing moving these checks closer to the source of the values. I'll do one more pass with the help of your comments to clean that up. Thanks!

Reply:
👍🏻